Nos/Service/RelayService.swift

```diff
@@ -690,6 +690,8 @@ extension RelayService: WebSocketDelegate {
             case .error(let error):
                 await subscriptions.remove(socket)
                 handleError(error, from: socket)
+            case .peerClosed:
```

Comment:
why did I have to add this to compile?

Reply:
Our web sockets library updated and the package rules were set to track `master`. I change it to a specific version though and I thought I already fixed this error 🤔